kubernetes/config-maps/http-configmap.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: http-configmap
+data:
+  check.digit.host: check-digit.default.svc.cluster.local
```

Comment:
What is using `check.digit.host` and `census.host` at the moment? The platform no longer has dependencies into census and check-digit. It'll be more clear if we can remove these from this config-map so there are no false assumptions.